chemprop/cli/common.py

```diff
@@ -28,7 +28,6 @@ def add_common_args(parser: ArgumentParser) -> ArgumentParser:
         action="store_true",
         help="Turn off using the first row in the input CSV as column names",
     )
-
```

Comment:
Did you intend to delete the lines in this file. I think we should keep the extra blank lines. If you didn't intend to delete them, it is possible that your `black` formatter did it automatically for you and could be reconfigured.